pkg/remoteimage/pull.go

```diff
@@ -28,6 +29,18 @@ type puller struct {
 	keyring  credentialprovider.DockerKeyring
 }
 
+// Returns the size of the image that was pulled in MB(I think?) **TODO: check**
+func (p puller) ImageSize(ctx context.Context) int {
+	imageSpec := &cri.ImageSpec{Image: p.image.String()}
+	return imageSpec.Size()
```

Comment:
This is not correct to get the size of an image. After checking the implementation of the [ImageSpec.Size](https://github.com/kubernetes/cri-api/blob/8a4a30b67ca093a927a0eadd20befd79e93fe29d/pkg/apis/runtime/v1alpha2/api.pb.go#L17085C18-L17085C18), I found that the Puller.ImageSize will always return the length of the image reference, that is, p.image here.